helm/hades/readme.md

```diff
@@ -0,0 +1,124 @@
+# Hades Helm Chart
+
+Deploy the **Hades** build system (API, Scheduler, and NATS broker) into any Kubernetes cluster using Helm.
+
+---
+
+## Contents
+
+* [Overview](#overview)
+* [Prerequisites](#prerequisites)
+* [Quick Start](#quick-start)
+* [Configuration](#configuration)
+
+    * [Values Reference](#values-reference)
+* [Upgrade & Rollback](#upgrade--rollback)
+* [Uninstall](#uninstall)
+* [Development](#development)
+
+---
+
+## Overview
+
+This chart bundles the core services of Hades:
+
+| Component           | Description                                                                      |
+|---------------------|----------------------------------------------------------------------------------|
+| **hades-api**       | Processes and validates the request and produce the build request as NATS events |
+| **hades-scheduler** | Consumes NATS events and spawns per-exercise runner Pods                         |
+| **hades-nats**      | Embedded [NATS JetStream](https://nats.io) message broker (sub-chart)            |
+
+The Scheduler operates in **`serviceaccount` mode** by default, using Kubernetes-native in-cluster credentials to authenticate and create jobs.
+
+> ⚠️ The previously supported `kubeconfig` mode is not implemented in this version.
+
+---
+
+## Prerequisites
+
+* Kubernetes **v1.25+**
+* Helm **v3.12+**
+
+---
+
+## Quick Start
+
+```bash
+# 1. Create target namespace if the namespace does not exist
+kubectl create namespace hades
+
+# 2. Install the chart using serviceaccount mode (default)
```

Comment:
There is also a missing step to update the values (there are some that have to be adapted - e.g. the hostname). Please at least mention this and update the next command to include the custom values during the install